x-pack/plugins/serverless_search/public/layout/nav.tsx

```diff
@@ -33,6 +33,10 @@ const navigationTree: NavigationTreeDefinition = {
       title: 'Elasticsearch',
       icon: 'logoElasticsearch',
       defaultIsCollapsed: false,
+      accordionProps: {
+        forceState: 'open',
+        arrowProps: { css: { display: 'none' } },
```

Comment:
```suggestion
        arrowDisplay: 'none',
```

Reply:
@cee-chen I tried this out, and I'm seeing the arrow still visible, yet moved to the left.

Before:
<img width="286" alt="image" src="https://github.com/elastic/kibana/assets/908371/992e74bd-11d3-4d22-9692-c3d8f479bce0">


After:
<img width="313" alt="image" src="https://github.com/elastic/kibana/assets/908371/c2a0aa55-6de6-4d6c-8c92-d2ceb82f8ed6">

Reply:
🤦 That's a bug in EUI, I'll get a fix out

Reply:
Ah apologies, it's not a bug, it's simply that accordions don't allow the arrow to be hidden if the button isn't interactive. Essentially, there always needs to be a way to toggle the accordion.

@ryankeairns 

> defaults on the solution nav group so that it is always open (i.e. has no collapse option/icon)?

Can you rethink this ask? If it's truly crucial, the component used essentially **can't** be an accordion. Hiding the arrow is fine for visual users but there's still a ton of accessibility implications of accordions that's going to confuse screen reader users if we leave it in.

If we need a way to render a header with items that's not collapsible, that's fine, but I need to render it a different way than using an underlying `EuiAccordion`.

Reply:
> If we need a way to render a header with items that's not collapsible, that's fine, but I need to render it a different way than using an underlying EuiAccordion.

10-4. I'll come back around to this with an EUI issue.

Reply:
Created this feature request in EUI: https://github.com/elastic/eui/issues/7208